client/app/scripts/liveblog-edit/views/posts.html

```diff
@@ -9,10 +9,11 @@
       index=$index
       posts-list-ctrl="postsList"
       start-reorder="postsList.startReorder(post)"
+      clear-reorder-action="postsList.clearReorder"
```

Comment:
Later, it would be nice to remove these parameters and call directly method through `postsListCtrl` defined just above.
Because these parameters are not really part of the directive configuration.